spec/features/mechanics/show_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'Mechanic Show Page', type: :feature do
+  before :each do
+    @park = AmusementPark.create!(name: 'Six Flags', admission_cost: 75)
+    @ride1 = @park.rides.create!(name: 'The Frog Hopper', thrill_rating: 5, open: true)
+    @ride2 = @park.rides.create!(name: 'Fahrenheit', thrill_rating: 8, open: true)
+    @new_ride = @park.rides.create!(name: 'New Ride', thrill_rating: 7, open: true)
+    @mechanic = Mechanic.create!(name: 'Kara Smith', years_experience: 11)
+    @mechanic.rides << @ride1
+    @mechanic.rides << @ride2
+  end
+
+  it 'shows the mechanic name, years of experience, and names of all rides they are working on' do
+    visit mechanic_path(@mechanic)
+
+    within('.mechanic-info') do
+      expect(page).to have_content(@mechanic.name)
+      expect(page).to have_content(@mechanic.years_experience)
+    end
+
+    within('.rides') do
+      expect(page).to have_content(@ride1.name)
+      expect(page).to have_content(@ride2.name)
+    end
```

Comment:
Since the new ride is not associated with the mechanic, asserting that the new ride doesn't show up on the page would have been another good assertion to add.